pom.xml

```diff
@@ -104,7 +103,7 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-failsafe-plugin</artifactId>
-                <version>3.0.0-M7</version>
+                <version>3.1.0</version>
```

Comment:
The latest verison is [3.2.5](https://mvnrepository.com/artifact/org.apache.maven.plugins/maven-failsafe-plugin).
For some reason inteleji sugestion hasn't updated that to 3.2.5 and latest version you can see is 3.1.0 I belive.